src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchOptionsFactoryTests.cs

```diff
@@ -468,6 +468,19 @@ public void GivenSetCoreFeatureForIncludeCount_WhenCreated_ThenSearchOptionsHave
             Assert.Equal(_coreFeatures.DefaultIncludeCountPerSearch, options.IncludeCount);
         }
 
+        [Fact]
+        public void GivenSearchParameter_Text_WhenCreated_ThenSearchParameterShouldBeAddedToUnsupportedList()
```

Comment:
Nit: the _ between Parameter and Text should be removed for consistency.

Reply:
> Nit: the _ between Parameter and Text should be removed for consistency.

Done

Reply:
This test name still has the extra _

Reply:
Fixed now. Sorry for that.